src/actions/attributes.ts

```diff
@@ -57,13 +57,13 @@ export function set_dataset_attribute(this: StreamElement) {
 }
 
 export function set_property(this: StreamElement) {
-  const name = this.getAttribute("name")
-  const value = this.getAttribute("value") || ""
+  const property = this.getAttribute("property")
+  const value = checkForBoolean(this.getAttribute("value") || "")
```

Comment:
I'm wondering if we should also consider a cast to `Number`. Or if the casting itself should be a separate attribute on the action itself. Something like `type="boolean"` or `type="number"` (also `array` and `object` if that makes sense).

What do you think?

Reply:
I'd vote for Turbo Power to do the casting. 

Seems like `JSON.parse()` is an easy way to do this but it has to be wrapped in a `try/catch` b/c it is so noisy.

I could expand on `checkForBoolean` (changing the name ofc) for the time being so it can be used in certain methods until a global change is made?

The attribute approach is interesting too because the user can dial in the type exactly so there are no surprises but in reality it'll be superfluous b/c auto casting would work as expected 99% of the time.

Either technique is going to require a good amount of work or at least changed files, therefore, it should be a new feature PR.

Reply:
If someone is running into this you can, in the meantime, use the `.invoke` method, which is (perhaps temporarily) exposed from the underlying library [TurboBoost Streams](https://github.com/hopsoft/turbo_boost-streams) e.g.:

`turbo_stream.invoke('checked =', args: [true], selector: '#order_form_any_time')`

Reply:
I think `JSON.parse()` is the right way here, I've put up #28 which adds a `typecast()` util function which should could come in handy here.

Reply:
Ok, I'll rework this PR after you merge in #28. Regarding this PR – Still changing the attribute to `property`?

Reply:
Thank you! 

I just merged #28. I'd like to keep the attribute as `name`, see: https://github.com/marcoroth/turbo_power/pull/27#discussion_r1059153516

Reply:
Oops I missed that comment. 

Got it, therefore, https://github.com/marcoroth/turbo_power-rails/pull/14

Reply:
This PR has been updated.